data/yesayajones.json

```diff
@@ -8,6 +8,11 @@
       "name": "Follow me on GitHub",
       "url": "https://github.com/yesayajones",
       "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/yesayajones",
+      "icon": "FaTwitter"
```

Comment:
Does this work? Shouldn't this be `twitter` only?

Reply:
This is correct, we now use the official icon names 👍 

https://linkfree.eddiehub.io/docs/quickstart